x/axelarcork/types/params.go

```diff
@@ -139,6 +139,10 @@ func validateExecutorAccount(i interface{}) error {
 		return errors.New("executor account cannot be empty")
 	}
 
+	if _, err := common.ParseHexOrString(execAcc); err != nil {
```

Comment:
I feel like we've been using a different address parser from the ethereum libs elsewhere in our modules, what's the difference with `ParseHexOrString`? I think one example is `IsHexAddress`.